src/Symfony/Component/Security/Http/HttpUtils.php

```diff
@@ -144,14 +144,13 @@ public function checkRequestPath(Request $request, $path)
     /**
      * Generates a URI, based on the given path or absolute URL.
      *
-     * @param Request $request A Request instance
```

Comment:
if it gets removed here, it should also be removed in the other methods like createRedirectResponse

Reply:
Done across the board in #32973